modules/tensor_mechanics/doc/content/source/utils/RankTwoScalarTools.md

```diff
@@ -0,0 +1,249 @@
+# Rank Two Scalar Tools
```

Comment:
This material looks really familiar. Is it copied from another location? Maybe the AuxKernels that use it? We should remove it from that other place and provide a link to this page from there if it is.